core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -171,7 +174,7 @@ class HttpGoogleServicesDAO(
     def insertBucket: (Map[WorkspaceAccessLevel, RawlsGroup], Option[Map[WorkspaceAccessLevel, RawlsGroup]]) => Future[String] = { (accessGroupsByLevel, intersectionGroupsByLevel) =>
       val bucketName = getBucketName(workspaceId)
       retryWhen500orGoogleError {
-        () => {
+        implicit count => {
```

Comment:
Look at `GoogleUtilities.scala`. :) The retry count and counters are passed implicitly to `executeGoogleRequest`, so I luckily didn't have to change too much in this class.